network/p2p/builder/libp2pNodeBuilder.go

```diff
@@ -487,7 +488,7 @@ func (b *LibP2PNodeBuilder) configureRoutingSystem(
 	} else {
 		// bitswap requires a content routing system. this returns a stub instead of a full DHT
 		b.SetRoutingSystem(func(ctx context.Context, host host.Host) (routing.Routing, error) {
-			return none.ConstructNilRouting(ctx, host, nil, nil)
+			return routinghelpers.Null{}, nil
```

Comment:
`none.ConstructNilRouting` was [removed](https://github.com/ipfs/boxo/blob/2fdde029d0bf4fe007a6a36b4ac0d5d02b88b765/CHANGELOG.md#removed-2). Using recommended alternative. The implementations differ in that the `none` version returned `nil` from `PutValue`, but the new version returns an error. Will need to see how this behaves with testing.